Engines/Wine/Verbs/FAudio/script.js

```diff
@@ -26,29 +28,16 @@ Wine.prototype.faudio = function (faudioVersion) {
         .extract();
 
     var forEach = Array.prototype.forEach;
-    var sys32dir = this.system32directory();
+    var sys64dir = this.system64directory();
     var faudioDir = this.prefixDirectory() + "/FAudio/faudio-" + faudioVersion;
     var self = this;
 
-    forEach.call(ls(faudioDir + "/x32"), function (file) {
+    forEach.call(ls(faudioDir + "/x64"), function (file) {
         if (file.endsWith(".dll")) {
-            cp(faudioDir + "/x32/" + file, sys32dir);
-            self.overrideDLL()
```

Comment:
You should keep dll overrides.

Reply:
How, you should also override the dlls like in the dxvk scrips ^^ (sorry, my bad):
```js
self.overrideDLL()
                .set("native", [file])
                .do();
```